db/migrate/20180704201742_drop_default_theme_key_from_site_setting.rb

```diff
@@ -0,0 +1,20 @@
+class DropDefaultThemeKeyFromSiteSetting < ActiveRecord::Migration[5.2]
+  def up
+    # preserve the current default theme by copying its ID to the new site setting...
```

Comment:
lets get all the work we need done, done in one migration no need for a ton here, very small amounts of data is changing.